src/main/java/edu/kis/powp/jobs2d/logger/UsageLogger.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.powp.jobs2d.logger;
+
+public class UsageLogger {
```

Comment:
Should be extended with observer. Use Publisher attribute (similarly to DriverCommandManager) or via inheritance.